src/main/java/mod/chiselsandbits/items/ItemPositivePrint.java

```diff
@@ -303,10 +320,25 @@ else if ( bit.isValid() )
 			}
 		}
 
+		//The state id of the last item in spawnlist.
+		int entityItemState = 0;
+
 		for ( final EntityItem ei : spawnlist )
 		{
 			ModUtil.feedPlayer( world, who, ei );
 			ItemBitBag.cleanupInventory( who, ei.getEntityItem() );
+			entityItemState = ItemChiseledBit.getStackState( ei.getEntityItem() );
```

Comment:
This seems like it would fail to present the message if some of the bits could fit, while others could not.